src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/ClaimStore.java

```diff
@@ -25,10 +24,7 @@
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
     @Autowired
-    private ClaimRepository claimRepository;
-
-    @Autowired
-    private LegacyCaseRepository legacyCaseRepository;
+    private CaseDBI caseDBI;
```

Comment:
where has DBI come from?
Its not a spring term as far as I'm aware,
we use repository